### ISSUE TITLE
Any chance of being able to add tasks?

### ISSUE THREAD

**Issue by johnhunt** [MarkTiedemann/anydo-cli#2]
I'd like this feature, however I'm not a node coder... I can have a go I guess?

**Comment by MarkTiedemann** [comment]
Yeah, sure, have a go at it! And let me know if you need any help, I'd be happy to help you out.

**Comment by MarkTiedemann** [comment]
To help you get started: This module is just a CLI module wrapping the API with a command line interface. So, to add this feature, you need to implement an API for it first. The API is located in its own repository at [MarkTiedemann/anydo](https://github.com/MarkTiedemann/anydo). Once you have implemented the API for letting you add tasks in that repository, you can add the CLI interface for it in this repository.
